packages/app/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json

```diff
@@ -1,7676 +1,7741 @@
-												{
-													"end": 7,
-													"data": {},
-													"type": "b",
-													"start": 0
-												}
-											]
-										}
-									}
-								]
-							},
-							"children": []
-						},
-						{
-							"id": "b338015a-54e6-488c-bd4d-c3b13aeebd3e",
-							"type": "ObojoboDraft.Chunks.Figure",
-							"content": {
-								"alt": null,
-								"url": "https://www.ucf.edu/files/2017/10/knightro_two_hands_point.png",
-								"size": "small",
-								"width": null,
-								"height": null,
-								"textGroup": [
-									{
-										"data": null,
-										"text": {
-											"value": "",
-											"styleList": []
-										}
-									}
-								]
-							},
-							"children": []
-						},
-						{
-							"id": "2dd8eda7-ae33-416a-81a9-c68fc1a9b8fb",
-							"type": "ObojoboDraft.Chunks.Text",
-							"content": {
-								"textGroup": [
-									{
-										"data": {},
-										"text": {
-											"value": "For more information on writing alt text, read the WebAIM article about alt text and accessibility. (Right-click on the link to open it)",
-											"styleList": [
-												{
-													"end": 99,
-													"data": {
-														"href": "https://webaim.org/techniques/alttext/"
-													},
-													"type": "a",
-													"start": 47
-												}
-											]
-										}
-									}
-								]
-							},
-							"children": []
-						},
-						{
-							"id": "792bf393-4623-4e6c-8159-66ceb826fbcd",
-							"type": "ObojoboDraft.Chunks.Heading",
-							"content": {
-								"textGroup": [
-									{
-										"data": {},
-										"text": {
-											"value": "Figure Sizes",
-											"styleList": []
-										}
-									}
-								],
-								"headingLevel": 3
-							},
-							"children": []
-						},
-						{
-							"id": "85c0ebaf-ae54-4d4e-a73c-12915faad38b",
-							"type": "ObojoboDraft.Chunks.Text",
-							"content": {
-								"textGroup": [
-									{
-										"data": {
-											"indent": 0
-										},
-										"text": {
-											"value": "The third property in the Image Properties dialog is the image size. Figures can be displayed in several different sizes: small, medium, large, and custom. ",
-											"styleList": [
-												{
-													"end": 128,
-													"data": {},
-													"type": "monospace",
-													"start": 123
-												},
-												{
-													"end": 136,
-													"data": {},
-													"type": "monospace",
-													"start": 130
-												},
-												{
-													"end": 143,
-													"data": {},
-													"type": "monospace",
-													"start": 138
-												},
-												{
-													"end": 155,
-													"data": {},
-													"type": "monospace",
-													"start": 148
-												}
-											]
-										}
-									}
-								]
-							},
-							"children": []
-						},
-						{
-							"id": "80522f9a-4b90-
```

Comment:
Just pointing out the typo "and then open the up the new page"

Also, is there any way we can emphasize the text for "+ Add Page" and "+ Add Assessment"? I think it might help with readability.